README.md

```diff
@@ -31,6 +31,11 @@ To override the default migrations path (`db/migrate`), add the following line t
 Mongoid::Migrator.migrations_path = ['foo/bar/db/migrate', 'path/to/db/migrate']
 ```
 
+To override the default rake task namespace to avoid collision with `ActiveRecord` rake tasks, add the following line to your `application.rb` file:
+```
+Mongoid::Migrator.namespace = 'db:mongoid'
+```
+
```

Comment:
Thanks for the readme part, I would probably some example rake commands after that, like: `rake db:mongoid:migrate` ?

Reply:
That's perfect, thanks!